P/PETSc/build_tarballs.jl

```diff
@@ -7,14 +7,13 @@ name = "PETSc"
 version = v"3.18.7"
 petsc_version = v"3.18.6"
 MUMPS_COMPAT_VERSION = "5.6.2"
-SUITESPARSE_COMPAT_VERSION = "7.2.1" 
 SUPERLUDIST_COMPAT_VERSION = "8.1.2"   
 MPItrampoline_compat_version="5.2.1"
 BLASTRAMPOLINE_COMPAT_VERSION="5.8.0"    
```

Comment:
I think you want this to be 5.4.0, and `julia_compat` to be `1.9` for using LBT.